pkgs/by-name/la/labelife-label-printer/package.nix

```diff
@@ -0,0 +1,55 @@
+{ lib, stdenv, fetchurl, cups, autoPatchelfHook }:
```

Comment:
```suggestion
{ lib
, stdenv
, fetchurl
, cups
, autoPatchelfHook
}:
```

Reply:
It's a simple derivation so I kept its function arguments compact and I don't anticipate needing to change them when (if ever) Labelife release a new version.

Reply:
I understand, and that make sense. But since most of the derivation in `nixpkgs` uses one line per parameter, maybe it would be nice to follow the trend.

Reply:
You did it well in the other PR https://github.com/NixOS/nixpkgs/pull/295600, why not maintaining consistency ?